python/samples/solutions/01-Semantic-Functions/README.md

```diff
@@ -0,0 +1,39 @@
+# Creating semantic functions
+
+This console application demonstrates the final solution to the [Creating semantic functions](https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/semantic-functions) doc article.
+
+## Prerequisites
+
+- [Python](https://www.python.org/downloads/) 3.8 and above
+  - [Poetry](https://python-poetry.org/) is used for packaging and dependency management
+  - [Semantic Kernel Tools](https://marketplace.visualstudio.com/items?itemName=ms-semantic-kernel.semantic-kernel)
```

Comment:
I don't think the Semantic Kernel Tools are a prerequisite for this project. It isn't used anywhere.

Reply:
```suggestion
```

Reply:
I now realize that these were generated from Semantic Kernel Tools :)